publishNewHelmChartVersion.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
```

Comment:
Should we rename the file as `update_helm_version_locally.sh` since the names confuses if the helm version is pushed on repo.

Reply:
I changed the file name